assets/templates/big-data-input.html

```diff
@@ -0,0 +1,467 @@
+                          name="name"
+                          id="name"
+                          placeholder="Lorem ipsum"
+                        />
+                        <div class="wrapper-errors dp-message dp-error-form">
+                          <p>¡Ouch! Este campo está vacío</p>
+                        </div>
+                      </li>
+                      <!-- utilizar la clase dp-validate en el li para mostrar mensajes de validacion -->
+                      <li class="field-item field-item--50">
+                        <div class="dp-wrapper-bd">
+                          <label for="lastname">*Asunto:</label>
+                          <input
+                            type="text"
+                            name="lastname"
+                            id="lastname"
+                            placeholder=""
+                          />
+
+                          <div class="dp-button-group">
+                            <div class="dp-bd-box">
+                              <button
+                                type="button"
+                                class="dp-button button-medium dp-button-bd"
+                                aria-controls="dp-options-cf"
+                              >
+                                <i class="ms-icon icon-brackets"></i>
+                              </button>
+                              <div class="dp-content-bd">
+                                <ul class="dp-list-cfield" id="dp-options-cf">
+                                  <li role="none">
+                                    <a role="menuitem">Nombre</a>
+                                  </li>
+                                  <li role="none">
+                                    <a role="menuitem">Apellido</a>
+                                  </li>
+                                  <li role="none">
+                                    <a role="menuitem">Cumpleaños</a>
+                                  </li>
+                                  <li role="none">
+                                    <a role="menuitem">Email</a>
+                                  </li>
+                                  <li role="none">
+                                    <a role="menuitem">Nombre</a>
+                                  </li>
+                                  <li role="none">
+                                    <a role="menuitem">Apellido</a>
+                                  </li>
+                                </ul>
+                              </div>
+                            </div>
+                            <div class="dp-bd-box">
+                              <button
+                                type="button"
+                                class="dp-button button-medium dp-button-bd"
+                                aria-controls="dp-menu-emojis"
+                              >
+                                <img src="img/emoji.png" />
```

Comment:
Why do you have the emoji here as png and not as an icon class?

Reply:
I check it with Manu!